src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -9,24 +17,56 @@
             public static final int xboxController = 1;
 
         }
-        public static class CANMotor {}
+        public static class CANMotor {
+            public static final int hatcherRollerIntake = 1; //Adjust port numbers
+            public static final int cargoRollerIntake = 2; //Adjust port numbers
+        }
         public static class PWM {}
         public static class CAN {}
-        public static class Pneumatics {}
+        public static class Pneumatics {
+
+           public static final PCMPort placeSolenoid = new PCMPort(1, 1, 1); //Adjust port numbers
+           public static final PCMPort pickupSolenoid = new PCMPort(3, 3, 3); // TODO: Adjust port numbers
+           public static final PCMPort flipper = new PCMPort(2, 2, 2); //Adjust port numbers
+
+        }
     }
     
     public static class Metrics{}
     public static class Component {
         public static CustomXbox driverXbox;
+
+        public static class Floorio {
+            public static DoubleSolenoid pickupSolenoid;
+            public static DoubleSolenoid placeSolenoid;
+            public static FloorIO floorio;
+            public static VelcroPlate velcroPlate;
+            public static FloorIO.Flipper flipper;
+            public static Motor cargoRollerIntake;
+            public static Motor hatcherRollerIntake;
+        }
     }
     public static class HumanInput {
         public static class Driver {
             public static CustomXbox xbox;
         }
-        public static class Operator {}
+        public static class Operator {
+            public static CustomJoystick joystick;
+        }
     }
+
     public RobotMap() {
         Component.driverXbox = new CustomXbox(Port.HumanInput.xboxController);
-		Component.driverXbox.setDeadZone(0.1);
+        
+        Component.Floorio.cargoRollerIntake = new Motor("Cargo Rolller Intake", new CANTalonSRX(Port.CANMotor.cargoRollerIntake));
+        Component.Floorio.hatcherRollerIntake = new Motor("Hatch Rolller Intake", new CANTalonSRX(Port.CANMotor.hatcherRollerIntake));
+
+        Component.Floorio.flipper = new FloorIO.Flipper(Port.Pneumatics.flipper.buildDoubleSolenoid());
+        //Component.Floorio.hatchHolder = new FloorIO.HatchHolder(Port.Pneumatics.hatchHolder.buildDoubleSolenoid());
```

Comment:
delete commented out code

Reply:
Change ALL port numbers to -1. In comments of what to change, write "TODO: " for all of them...